user_tools/src/spark_rapids_pytools/cloud_api/onprem.py

```diff
@@ -38,6 +38,7 @@ class OnPremPlatform(PlatformBase):
     def __post_init__(self):
         self.type_id = CspEnv.ONPREM
         self.platform = self.ctxt_args.get('targetPlatform')
+        self.cluster_inference_supported = False
```

Comment:
Why we do not support cluster_inference for onprem?

Reply:
I see. We need to discuss if the user is going to pass that argument or not.
Pros: It gives user flexibility
Cons: If we are aiming at fine granularity, then we cannot ask the user to pass a cluster for each app.

So usability wise, it seems that we should drop that input. But again, let's not make a change for the PR for now.

Reply:
Resolving this comment as we are not making a change for this PR

Reply:
Ok